src/vmm/src/arch/x86_64/msr.rs

```diff
@@ -49,7 +49,7 @@ const APIC_BASE_MSR: u32 = 0x800;
 /// Number of APIC MSR indexes
 const APIC_MSR_INDEXES: u32 = 0x400;
 
-/// Custom MSRs fall in the range 0x4b564d00-0x4b564dff
+/// Custom KVM MSRs fall in the range 0x4b564d00-0x4b564dff
```

Comment:
Is this change left over from the PVM MSR changes?

Reply:
Nope